common/test/acceptance/pages/lms/discussion.py

```diff
@@ -463,3 +463,25 @@ def _match_messages(text):
             lambda: _match_messages(text).results == [],
             "waiting for dismissed alerts to disappear"
         ).fulfill()
+
+    def click_new_post_button(self):
```

Comment:
Some similar methods are being added in https://github.com/edx/edx-platform/pull/6034. To avoid merge conflicts, one implementation of these methods needs to be chosen.